homework/12_V/11_Georgi_Stanachkov/homework1/main.py

```diff
@@ -0,0 +1,28 @@
+import math
+import re
+
+
+def find_relative_position(x1, y1, r1, x2, y2, r2):

```

Comment:
Instead of using `1, 2, 3` to indicate the different conditions can you refactor the code to use an enum?